src/dmd/expressionsem.d

```diff
@@ -6287,6 +6287,42 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
         result = e.expressionSemantic(sc);
     }
 
+    /*
+     * Get the expression initializer for a specific struct
+     *
+     * Params:
+     *  sd = the struct for which the expression initializer is needed
+     *  loc = the location of the initializer
+     *  sc = the scope where the expression is located
+     *  t = the type of the expression
+     *
+     * Returns:
+     *  The expression initializer
```

Comment:
I think this should mention that it might return `null` and why.

Reply:
Or, better yet, shouldn't this return an `ErrorExp` and then check for `TOK.error` in the caller?